### PR TITLE
[UXP-22719][UXP-22720] - Added support of SWC sp-action-menu & sp-picker (v0.37.0) in swc-uxp-wrapper for UXP

### PR DESCRIPTION
**Pull request by inasirhussain**
Closes - 
https://jira.corp.adobe.com/browse/UXP-22719
https://jira.corp.adobe.com/browse/UXP-22720
https://jira.corp.adobe.com/browse/UXP-21368

We've overridden the `PickerBase `render method to address UXP's native button customisation limitation, where SWC's `PickerBase` code expects to add an error icon to the right end of the button. Also positioning of content inside UXP's native button is always centre aligned whereas SWC's `PickerBase` expect it to be left aligned. Considering these issue we have replaced native html `button` with a `div` and handled the subsequent code in `swc-uxp-wrappers`

Since we have overridden the `PickerBase`, have to override the `Picker` code as well in `swc-uxp-wrappers`

### CODE REVIEW

File: packages/utils/package.json [hunk 1]
```diff
@@ -18,7 +18,8 @@
     "exports": {
         ".": "./src/index.js",
         "./package.json": "./package.json",
-        "./src/aliases.js": "./src/aliases.js"
+        "./src/aliases.js": "./src/aliases.js",
+        "./src/MatchMediaController.js": "./src/MatchMediaController.js"
```
Comment (deepthi-majji) [inline packages/utils/package.json:22]: Can't we add this line in aliases.js?

File: packages/utils/src/MatchMediaController.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
```
Comment (deepthi-majji) [inline packages/utils/src/MatchMediaController.js]: nit: 2024

File: packages/utils/package.json (hunk 1 above)
Comment (deepthi-majji) [reply packages/utils/package.json:22]: okay got it.

File: packages/popover/src/uxp-popover.css [hunk 3]
```diff
@@ -34,3 +34,7 @@ governing permissions and limitations under the License.
     height: 100%;
     width: 10px;
 }
+
+::slotted(.visually-hidden) {
```
Comment (deepthi-majji) [inline packages/popover/src/uxp-popover.css]: Why wasn't this working earlier?

File: packages/utils/src/MatchMediaController.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+// Custom  MatchMediaController
+export function MatchMediaController(context, isMobile) {
+    return false;
```
Comment (deepthi-majji) [inline packages/utils/src/MatchMediaController.js:22]: Can you add the original spec?

File: projects/swc-starter-webpack/src/example_usages/sp-action-menu.html [hunk 5]
```diff
@@ -0,0 +1,279 @@
...
+                        <sp-menu-item>Select and mask...</sp-menu-item>
+                        <sp-menu-divider></sp-menu-divider>
+                        <sp-menu-item>Save selection</sp-menu-item>
+                        <sp-menu-item disabled>Make work path</sp-menu-item>
+                    </sp-action-menu>
+                </div>
+            </div>
+            <div id="variantLabel">open</div>
+            <div class="demo-example" style="display: flex">
+                <sp-action-menu size="m" open>
```
Comment (deepthi-majji) [inline projects/swc-starter-webpack/src/example_usages/sp-action-menu.html]: This is highly unlikely that one would have open to be true by default and there is a bug in this in UXP. Lets remove this.

File: packages/picker/src/Picker.js [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+        `;
+    }
+}
+
+export { UxpPickerBase as PickerBase };
+
+class UxpPicker extends UxpPickerBase {
+    constructor() {
+        super(...arguments);
+        this.handleKeydown = (event) => {
```
Comment (deepthi-majji) [inline packages/picker/src/Picker.js:60]: please add the reason in PR description.

File: packages/popover/src/uxp-popover.css (hunk 3 above)
Comment (inasirhussain) [reply packages/popover/src/uxp-popover.css]: Removed, this isn't needed.

File: packages/picker/src/Picker.js (hunk 6 above)
Comment (deepthi-majji) [reply packages/picker/src/Picker.js:60]: Should it be keydown or keyup?

Comment (inasirhussain) [reply packages/picker/src/Picker.js:60]: its keydown, on the similar line to SWC code and i have tested this code works in PS

**Review by deepthi-majji — APPROVED** [review]

**Pull request merged**